models/spring-ai-mistral-ai/src/test/java/org/springframework/ai/mistralai/MistralAiRetryTests.java

```diff
@@ -156,6 +158,7 @@ public void mistralAiChatStreamTransientError() {
 	}
 
 	@Test
+	@Disabled("Currently stream() does not implement retry")
```

Comment:
It will be implemented in https://github.com/spring-projects/spring-ai/issues/1193